frontend/packages/pipelines-plugin/src/components/pipelineruns/utils/tekton-results.ts

```diff
@@ -0,0 +1,335 @@
+import {
+  K8sResourceCommon,
+  MatchExpression,
+  MatchLabels,
+  Selector,
+} from '@console/dynamic-plugin-sdk/src';
+import { RouteModel } from '@console/internal/models';
+import { k8sGet } from '@console/internal/module/k8s';
```

Comment:
Dynamic plugin sdk has a similar `k8sGetResource` function, so that we don't need the internal package here.